SPECS/systemd/systemd.spec

```diff
@@ -890,6 +890,11 @@ ln -s --relative %{buildroot}%{_bindir}/kernel-install %{buildroot}%{_sbindir}/i
 # Split files in build root into rpms
 python3 %{SOURCE2} %buildroot %{!?want_bootloader:--no-bootloader}
 
+%if 0%{?want_bootloader}
+mkdir -p %{buildroot}/boot/efi/EFI/BOOT
+cp %{buildroot}/usr/lib/systemd/boot/efi/systemd-bootx64.efi %{buildroot}/boot/efi/EFI/BOOT/grubx64.efi
```

Comment:
Where does the requirement come from for sd-boot to masquerade as grub?

Reply:
default loader is hardcoded to be grubx64.efi on x86-64

Reply:
https://github.com/rhboot/shim/blob/shim-15.3/shim.h#L69